frontend/catalyst/jax_extras/tracing.py

```diff
@@ -460,6 +457,17 @@ def deduce_avals(f: Callable, args, kwargs):
     return wffa, in_avals, keep_inputs, out_tree_promise
 
 
+def trace_to_jaxpr(
+    trace: DynamicJaxprTrace, inputs: List[DynamicJaxprTracer], outputs: List[DynamicJaxprTracer]
+) -> Tuple[Jaxpr, List[DynamicJaxprTracer], List[Any]]:
+    """Get Jaxpr from a Jax trace applying a workaround.  The workaround makes it possible to trace
+    e.g. the following program: `lambda i, a: i<3`. Where `a` is an unused dynamically-shaped array.
+    """
```

Comment:
```suggestion
```suggestion
    This method would remove return values related to sizes of tensors
    when compiling with dynamically sized tensors.
    """
```

Reply:
I think your suggestion is misformatted